crates/red_knot_python_semantic/resources/mdtest/type_properties/is_disjoint_from.md

```diff
@@ -80,6 +80,16 @@ static_assert(is_disjoint_from(tuple[Literal[1], Literal[2]], tuple[Literal[1],
 static_assert(not is_disjoint_from(tuple[Literal[1], Literal[2]], tuple[Literal[1], int]))
 ```
 
+If a tuple type contains a `Never` element, it is considered similar to `Never` itself and is
+disjoint from any other type.
+
+```py
+static_assert(is_disjoint_from(tuple[Never], tuple[Never]))
+static_assert(is_disjoint_from(tuple[int, Never], tuple[int, Never]))
+static_assert(is_disjoint_from(tuple[int, Never], tuple[Never, int]))
+static_assert(is_disjoint_from(tuple[int, Never], tuple[int, int]))
+```
+
```

Comment:
This is not related to this PR but it helped my understanding when I discussed `Never` usage with Alex and David last week so I added some test cases for it as they were missing.